linux/LinuxProcessList.c

```diff
@@ -1616,8 +1617,18 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
       if (! LinuxProcessList_updateUser(pl, proc, procFd))
          goto errorReadingProcess;
 
-      if (!LinuxProcessList_readStatusFile(proc, procFd))
-         goto errorReadingProcess;
+      if (ss->flags & PROCESS_FLAG_LINUX_CGROUP || hideRunningInContainer)
+         LinuxProcessList_readCGroupFile(lp, procFd);
```

Comment:
Does this mean that in the `hideRunningInContainer` case we trade reading `status` for reading cgroup? Is that supposed to be better?
Also, are the paths being checked systemd-specific?